queries.sql

```diff
@@ -150,4 +150,4 @@ WHERE animals.name = (
   WHERE vet = 'Maisy Smith'
   GROUP BY animal
   ORDER BY COUNT(*) DESC LIMIT 1
-);
\ No newline at end of file
+);
```

Comment:
- [ ] It would be better if our PR is more detailed for example. We can mention the tables we joined instead of just writing “Joined table” as a PR